Create numpy-accessibility.md (#12)
numpy/numpy-newsletter: numpy-feature/numpy-accessibility.md

---
title: "Making Numpy accessible"
---

Numpy is now foundational to Python scientific computing. Our efforts reach millions of developers each month. 
As our user base grows, so does the number of users with disabilities. In the open-source ecosystem, accessibility 

> is often considered low-priority, scary, or someone else’s job. Read the blog post by Mars Lee, a front end developer and designer at Quansight, 

Reviewer: ```suggestion
is often considered low-priority, scary, or someone else’s job. Read the blog post by Mars Lee, a front-end developer and designer at Quansight, 
```

> is often considered low-priority, scary, or someone else’s job. Read the blog post by Mars Lee, a front end developer and designer at Quansight, 

Author: ```suggestion
is often considered low-priority, scary, or someone else’s job. Read the blog post by Mars Lee, a front-end developer and designer at Quansight, 
```

> how she along with Melissa Weber Mendonça championed the efforts to support NumPy users with visual impairments 

Reviewer: ```suggestion
about how she and Melissa Weber Mendonça championed the efforts to support NumPy users with visual impairments 
```

> and make the NumPy website and documentation more accessible: https://labs.quansight.org/blog/2021/07/numpy-accessibility-guidelines/.

Reviewer: ```suggestion
and make the NumPy website and documentation more accessible: [*Making Numpy Accessible: Guidelines and Tools*](https://labs.quansight.org/blog/2021/07/numpy-accessibility-guidelines/)
```
